gitian-building.md

```diff
@@ -80,17 +80,21 @@ Where `satoshi` is your Github name and `0.16.0rc1` is the most recent tag (with
 In order to sign gitian builds on your host machine, which has your PGP key, fork the gitian.sigs repository and clone it on your host machine:
 
 ```
+export NAME=satoshi
 git clone git@github.com:bitcoin-core/gitian.sigs.git
-git remote add satoshi git@github.com:satoshi/gitian.sigs.git
+git remote add $NAME git@github.com:$NAME/gitian.sigs.git
```

Comment:
The second $NAME should be where the repo was forked to? Might want to clarify that in a follow-up?